configs/nginx/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx
+  namespace: nginx
+data:
+  threads: "{{ .Values.threads }}"
+  connections: "{{ .Values.connections }}"
+  duration: "{{ .Values.duration }}"
+  nginxIterations: "{{ .Values.nginxIterations }}"
+  pushgatewayURL: "{{ .Values.pushgatewayURL }}"
+  cloud: "{{ .Values.cloud }}"
+  instance: "{{ .Values.instance }}"
```

Comment:
Again the instance got the `-1` appended. 
![Uploading Screenshot from 2021-05-03 14-54-03.png…]()

Reply:
Oh I changed the `instance` value to `foobar` and I don't see `foobar` in the results. Something is wrong with the instance, maybe due to conflict.